packages/cli/tests/build.test.js

```diff
@@ -217,28 +222,8 @@ describe('preact build', () => {
 		expect(existsSync(file)).toBe(true);
 	});
 
-	it('should inject preact.* variables into template', async () => {
-		let dir = await subject('custom-template-2');
-		await build(dir);
-
-		let file = join(dir, 'build/index.html');
-		let html = await readFile(file, 'utf-8');
-
-		looksLike(html, images.templateReplaced);
-	});
-
-	it('should replace title with <%= preact.title %>', async () => {
-		let dir = await subject('custom-template-3');
-		await build(dir);
-
-		let file = join(dir, 'build/index.html');
-		let html = await readFile(file, 'utf-8');
-
-		looksLike(html, images.templateReplaced);
-	});
-
```

Comment:
I combined these tests into one. Should check all the same things, but it'll be slightly faster/less busy.